src/app/media/[id]/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { getMediaDetails } from "@/api/media";
+import Image from "next/image";
+import PlayIcon from "#/icons/home/play.svg";
+
+interface Params {
+  id: number;
+}
+
+interface DetailPageProps {
+  params: Params;
+}
+
+const DetailPage = async ({ params }: DetailPageProps) => {
+  const posterBaseUrl = process.env.NEXT_PUBLIC_POSTER_BASE_URL;
+  const data = await getMediaDetails(params.id);
+  return (
+    <div className="flex-center flex-column gap-32pxr">
+      <div className="relative h-415pxr w-full">
+        <Image src={`${posterBaseUrl}${data.poster_path}`} alt={`영화 포스터 이미지: ${data.title}`} fill />
+        <div className="absolute inset-0pxr bg-gradient-to-b"></div>
+      </div>
+      <button className="flex-center h-45pxr w-303pxr gap-15pxr rounded-md bg-gray-30 text-18pxr font-semibold text-black">
+        <PlayIcon alt="play 아이콘" />
+        Play
+      </button>
```

Comment:
Play 버튼을 컴포넌트로 분리하고 해당 컴포넌트를 파악하기 위해 중요한 정보인 "Play" 텍스트를 외부에서 넘겨주면 가독성에 더 좋을 것 같아요!!

또 비슷한 디자인의 Play 버튼이 홈에도 있어서 PlayButton 컴포넌트를 생성한 뒤, 세부 디자인은 외부에서 전달해서 변경하면 더 좋을 것 같네요!!(twMerge 라이브러리 사용)

PlayButton 컴포넌트 ->
```
   return <button className={twMerge("flex-center rounded-md bg-gray-30 text-18pxr font-semibold text-black", className)}>
      <PlayIcon alt="play 아이콘" />
      {text}
   </button
```


PlayButton 컴포넌트 사용(홈에서는 className 세부 디자인만 다르게 넘기기) ->
```suggestion
<PlayButton text="Play" className="h-45pxr w-303pxr gap-15pxr" />
```